src/scripts/fixtures_script.sql

```diff
@@ -0,0 +1,40 @@
+-- FUNCTION : product_fixture
+
+CREATE PROCEDURE insert_product_with_default(
+    		    				p_name varchar(50) default 'test_product',
```

Comment:
Remove `p_` this is unnecessary here, we already know we are going to insert a product.

Reply:
done.